commons/all/src/main/java/org/infinispan/commons/util/Version.java

```diff
@@ -162,6 +164,10 @@ public static short getVersionShort(String versionString) {
       return encodeVersion(a, b, c);
    }
 
+   public static String getUpstreamVersion() {
```

Comment:
It's ok to me, but having any reference to redhat in the code is accepted ? @tristantarrant

Reply:
I'd much rather have a `getUnbrandedVersion()` method. The properties file should contain two properties and it should be up to the productization script to handle the values, not the code.